pages/_app.js

```diff
@@ -14,6 +16,9 @@ export default function App({ Component, pageProps }) {
       <SWRConfig value={{ fetcher }}>
         <GlobalStyle />
         <Component {...pageProps} />
+        <Flex justifyContent="center">
+          <Nav />
+        </Flex>
       </SWRConfig>
     </main>
```

Comment:
I suggest to use a `Layout `component to clean up a bit 🧹 

Then your `_app.js` would look like:

```suggestion
    <SWRConfig value={{ fetcher }}>
      <Layout>  
        <GlobalStyle />
        <Component {...pageProps} />
      </Layout>
    </SWRConfig>
```

And your `Layout` component could look something like this:

```jsx
 <>
      <Head>
        <title>Star Wars Bazar</title>
      </Head>
      <main className={blinker.className}>{children}</main>
       <Flex justifyContent="center">
          <Nav />
        </Flex>
    </>
```

You can also have a look at the art gallery app again as the layout component was used there as well.